docs/website/src/site/index.md

```diff
@@ -51,4 +51,120 @@ eleventyNavigation:
+</svg>
+
+</div>
+
+<div class="ds-heading-2">
+<a href="/categories/design-tokens/" aria-label="Design tokens. Tokens are platform-agnostic variables that represent the look and feel of our brands.">
+Design tokens
+</a>
+</div>
+
+Tokens are platform-agnostic variables that represent the look and feel of our brands.
+
+</div>
+
+<div class="ds-explore__guidelines ds-explore__card">
+
+<div class="ds-explore__card-image">
+
+<svg width="144" height="140" viewBox="0 0 144 140" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M93.7651 119.762C89.3458 115.199 88.6092 109.116 91.5554 105.694L110.706 85.9243C114.02 82.5025 119.913 83.6431 124.332 88.2054L139.799 104.174C144.219 108.736 144.955 114.819 142.009 118.241L122.859 138.011C119.544 141.433 113.652 140.292 109.233 135.73L93.7651 119.762Z" fill="#FF575C"/>
+<path d="M126.959 91.0104L119.408 98.8066L47.2751 24.3392L54.8269 16.543L126.959 91.0104Z" fill="#45CC80"/>
+<path d="M121.604 96.5962L99.449 119.468L22.1082 39.6243L44.2634 16.7521L121.604 96.5962Z" fill="#45CC80"/>
+<path d="M103.736 114.633L96.1847 122.429L24.0521 47.9619L31.6038 40.1657L103.736 114.633Z" fill="#45CC80"/>
+<path d="M8.32544 0.000366211L13.4813 15.9685L23.7929 5.70328L8.32544 0.000366211Z" fill="#5985FF"/>
+<path d="M54.7279 16.3486L23.7929 5.70312L13.4812 15.9684L24.1611 48.2849L27.8439 44.1028L37.0507 45.6235L34.4728 26.994L52.5182 29.6554L51.0451 20.5307L54.7279 16.3486Z" fill="#FFD6D1"/>
+<path class="ds-explore__guidelines-part" d="M66.5125 53.9879C41.4699 79.8412 41.4699 91.6272 48.4671 99.2311C55.4643 106.455 66.8808 106.835 92.2917 80.6015C117.334 54.7483 146.796 12.5468 139.799 4.94287C132.434 -2.28082 91.5551 28.1347 66.5125 53.9879Z" fill="#FFD6D1"/>
+<path class="ds-explore__guidelines-part" d="M0.223389 130.027C8.32542 138.391 40.7335 141.052 56.201 124.704C57.6741 123.183 59.1472 120.902 60.252 119.001C35.946 118.241 23.0564 109.876 16.4274 103.413C13.8495 111.777 11.2716 122.423 0.223389 130.027Z" fill="#45CC80"/>
+<path class="ds-explore__guidelines-part" d="M58.4106 88.9656C49.572 79.841 32.9997 81.7419 23.7929 90.8666C20.1101 94.6686 18.2688 98.4705 16.7957 103.033C23.0563 109.496 35.5776 129.266 60.6203 118.621C65.7761 109.496 65.4078 96.5695 58.4106 88.9656Z" fill="#5985FF"/>
+<path class="ds-explore__guidelines-part" d="M16.7957 103.413C16.7957 103.413 31.5266 100.372 38.5238 111.397C45.521 122.043 60.252 119.001 60.252 119.001L41.47 130.027L22.3198 115.579L16.7957 103.413Z" fill="#45CC80"/>
+</svg>
+
+</div>
+
+<div class="ds-heading-2">
+<a href="/categories/guidelines/" aria-label="Guidelines. Practical guides to get started with designing and developing experiences with Sanoma Learning’s Design System.">
+Guidelines
+</a>
+</div>
+
+Practical guides to get started with designing and developing experiences with Sanoma Learning’s Design System.
```

Comment:
"your experience" instead of "experience", again, to involve the reader

Reply:
@Diaan, Thanks!
@anna-lach, Can we change it to:'Practical guides to get started with designing and developing your experience with Sanoma Learning's Design System'